qiskit/circuit/__init__.py

```diff
@@ -263,6 +263,8 @@
    AncillaRegister
    AncillaQubit
    CircuitInstruction
+   Register
+   Bit
```

Comment:
We refer to these times kinda often. Maybe we should include them in the doc.

Reply:
I think this is probably correct, though maybe if we're doing that we should add a couple of lines to either of their docstrings saying that they're effectively ABCs and shouldn't be instantiated directly.

Reply:
note added in decf87934bc7d03be451dc32a10d9144e9e33caf